pinery-miso/src/main/java/ca/on/oicr/pinery/lims/miso/MisoRunPosition.java

```diff
@@ -0,0 +1,26 @@
+package ca.on.oicr.pinery.lims.miso;
+
+import ca.on.oicr.pinery.lims.DefaultRunPosition;
+
+public class MisoRunPosition extends DefaultRunPosition {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Override the "equals" method in this class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2160)